### DISCUSSION TITLE
First Research Community Hackathon / Collabothon: 3rd June 2023

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#466] [Research]
Organize a [research community](https://lifeitself.org/research) hackathon in the next 3-4 months. Hackathons are a great pattern we could adopt as a way of crystallising activity and community.

> A hackathon (also known as a hack day, hackfest, datathon or codefest; a portmanteau of hacking and marathon) is an event where people engage in rapid and collaborative engineering over a relatively short period of time such as 24 or 48 hours

**UPDATE 📣** (28 April) date chosen for the event **Saturday and/or Sunday 3rd/4th of June**.

### Purpose and principles

Have a hackathon where a) existing team have work time together to ship something b) new people can get involved and connect 

- In person at Bergerac and online
- Contributing to and aligned with the draft research program of the community

### Outcome visioning

10+ people having a great time, 5 projects and we ship at least 3 published reading lists and have the infrastructure for doing more in ongoing collaborations.

### 📣 Suggestions wanted!

We're brainstorming around the following questions (and please add more**

- **What (topics)?**: 🚧 **see brainstorm below 👇**
- **Who**: all welcome!
- **Facilitators**: Rufus and Matt volunteered to co-organize. Others welcome.
- **Where**: online and at [Bergerac Hub](https://lifeitself.org/hubs/bergerac) or nearby
- **When**: Sat/Sun 3rd and 4th June 2023  ~~probably a weekend some time in next 3 months (i.e. may / june / july 2023)~~
- **Naming**: this isn't really a hackathon (we aren't coding and we aren't rushing to get somewhere!). Alternative names wanted!

### Topic shortlist

Culled from suggestions below

- **High-level knowledge map**: An interactive knowledge map viz that ties together topics of focus within Life Itself.
  - Attempting to serve the Q: _"What topics is Life Itself interested in?"_ and _"What topics are adjacent or related to X?"_.
  - Inspiration: [Life Itself research topical map](https://github.com/life-itself/community/blob/914addc2405ff1e7eff1d3f28e6db6ed87a8240c/content/excalidraw/research-topical-map-2023-01-14.excalidraw.svg) / [xiq's knowledge map](https://pbs.twimg.com/media/Fa2M570WIAA7USi?format=png&name=4096x4096)
- **Reading lists / learning resource curation**: A framework for organising learning resources for newcomers to a particular topic, i.e. books, papers, videos, podcasts, thinkers, etc. This could also be integrated with a knowledge map.
  - Attempting to serve the Q: _"How can I learn about topic X as a newcomer?"._ and _"What are the canonical texts for topic X?"_
  - Inspiration: [user-generated reading lists](https://placesjournal.org/reading-lists/) / [notion reading lists](https://parameter.io/notion-reading-list-templates/) / [roadmap.sh](https://roadmap.sh/system-design) / [healthygamer.gg](https://user-images.githubusercontent.com/960919/234788298-d2ef94a1-c7ae-4f7a-b108-19128f448cc1.png)
- how to analyze data (surveys, interviews before and after coliving experiences) quantitatively or qualitatively, e.g. some of the notebook example is here https://github.com/Liyubov/community/tree/master/research/notebooks we started to discuss it with Marc as well, as he has experience in quantiative measures, survey design etc.
- related to space of coliving, we started to also map coliving space e.g. here https://github.com/Liyubov/community/blob/master/research/README.md based on discussions we had with Penny Clark from CC (consious coliving), happy to invite her for this
- Inner development guide / directory could include library of papers, literature review, visualization of key streams of work
- Textual analysis of political manifestos to analyze their cultural worldview
- Write up of cultural values work by e.g. Schwartz, Inglehart et al
- Guide to conscious communities and coliving
- Directory of exemplar intentional communities
- Prototype research program about intentional communities and what makes them work or not

**Comment by Liyubov** [comment]
Hi @rufuspollock 
Topic wise some of the more concrete questions I could bring are related to quantitative analysis of community living, e.g. 

- how to analyze data (surveys, interviews before and after coliving experiences) quantitatively or qualitatively, e.g. some of the notebook example is here https://github.com/Liyubov/community/tree/master/research/notebooks we started to discuss it with Marc as well, as he has experience in quantiative measures, survey design etc.
- related to space of coliving, we started to also map coliving space e.g. here https://github.com/Liyubov/community/blob/master/research/README.md based on discussions we had with Penny Clark from CC (consious coliving), happy to invite her for this

**Reply by rufuspollock** [reply]
@Liyubov these are great suggestions - really like them.

And please do invite Penny Clark - who i know well!

**Comment by baouroux** [comment]
Ooops, I'm just catching up on this thread - from our last Friday call I had noted this as an action for me to post to the Forum, and created (with delay!) this thread this morning: https://github.com/orgs/life-itself/discussions/477
So a duplication.... I'll plan to copy my contribution to this thread and delete mine.

**Comment by rufuspollock** [comment]
Throwing out some ideas for topics:

- Inner development guide / directory could include library of papers, literature review, visualization of key streams of work
- Textual analysis of political manifestos to analyze their cultural worldview
- Write up of cultural values work by e.g. Schwartz, Inglehart et al
- Guide to conscious communities and coliving
- Directory of exemplar intentional communities
- Prototype research program about intentional communities and what makes them work or not

**Comment by mattosborn** [comment]
A couple more ideas around knowledge mapping that I'd be keen to explore. & would also be interesting to think about whether any of it could be done using collective approaches
  - **High-level knowledge map**
    An interactive knowledge map viz that ties together topics of focus within Life Itself.
    Attempting to serve the Q: _"What topics is Life Itself interested in?"_ and _"What topics are adjacent or related to X?"_.
    Inspiration: [Life Itself research topical map](https://github.com/life-itself/community/blob/914addc2405ff1e7eff1d3f28e6db6ed87a8240c/content/excalidraw/research-topical-map-2023-01-14.excalidraw.svg) / [xiq's knowledge map](https://pbs.twimg.com/media/Fa2M570WIAA7USi?format=png&name=4096x4096)
  - **Reading lists / learning resource curation**
    A framework for organising learning resources for newcomers to a particular topic, i.e. books, papers, videos, podcasts, thinkers, etc. This could also be integrated with a knowledge map.
    Attempting to serve the Q: _"How can I learn about topic X as a newcomer?"._ and _"What are the canonical texts for topic X?"_
    Inspiration: [user-generated reading lists](https://placesjournal.org/reading-lists/) / [notion reading lists](https://parameter.io/notion-reading-list-templates/) / [roadmap.sh](https://roadmap.sh/system-design) / [healthygamer.gg](https://user-images.githubusercontent.com/960919/234788298-d2ef94a1-c7ae-4f7a-b108-19128f448cc1.png)

On dates, I'd say June or July so there's at least a month for people to seed ideas, group around them and do some preliminary work/planning. Reason being that hackdays can work a bit smoother when starting the day with a clearer plan/intention, otherwise there's a danger of the day getting swallowed up with brainstorming and discovery.

And a rough schedule for the day could look something like:
- 09:00 – Kick-off and intro from each team about their project/plan
- 10:00 – Hacking
- 13:15 – Quick check-in
- 13:30 – Lunch
- 14:30 – Hacking
- 18:00 – Project presentations, feedback, Qs, discussion
- 19:00 – End

**Reply by rufuspollock** [reply]
> High-level knowledge map

> Reading lists / learning resource curation

Both really great suggestions Matt. I like these are both more "introductory" and support others engaging with this.

> And a rough schedule for the day could look something like:

Structure of day looks spot on.

**Comment by rufuspollock** [comment]
Following the research group meeting last Friday we are including towards Saturday and/or Sunday 3rd/4th of June.

**Reply by baouroux** [reply]
Great, sounds good! I'll be able to join you in Bergerac if there is an in-person option....
Also, very happy to offer for an in-person location near Toulouse our renovated farmhouse with 4 bedrooms (playroom could be a 5th room) and a large 75m2 open-space. We're 25min from Toulouse airport, 35min (without traffic) from Toulouse center.

**Reply by rufuspollock** [reply]
There will be an in person option so great to have you in person 🎉

**Comment by baouroux** [comment]
A couple of spin-offs and connections to follow-up on from the Hackathon:

@msantolini and @Liyubov

I've just gone through your amazing work during the hackathon, and followed a few of the related document links:

I'd love to 'stick my nose into' (!) your work on: 

- [Community maps Life Itself 2021 for editing](https://docs.google.com/document/d/1wy4fFZ-ympMCR_FeMrWTs_Dz8buWgABRBrVrSj203tE/edit)
- [Research and surveys for shared spaces: Bergerac/coliving](https://docs.google.com/document/d/1uQngQENHNLmDELzmflzm_m0nizCyi7uFqhha1P6aPiY/edit?usp=sharing)

These both resonate closely to my past Sensemaking consulting work that I have now rebranded to [www.collectivenavigator.com](http://www.collectivenavigator.com/).

I've put this brand on hold as I go deeper into my art (www.tyler.world) and narrative video projects ([www.interiortruth.com](http://www.interiortruth.com/)), and so my narrative software platform/hosting licences have since expired.

But I do see many opportunities to relaunch my larger-scale narratives work, and have been collaborating with the Princeton Engineering Anomalies Research Lab (PEAR) spin-off at https://gowyrd.org/research/ (for whom I've done some filming work), to draw up a funding proposal [Wyrd Tech - Narratives Capture Proposal](https://docs.google.com/document/d/18O8WmS70TujbXP143p5Z8xfgOcvv1j09z-TIzT45sjc/edit?usp=sharing) (perhaps for Templeton?) for collecting real time experiences of group contemplative experiences - for narrative correlation analysis with the PEAR Random Number Generator 'field effect' capture.

I mentioned to Marc the possible synchronisation with his concept of a "**Documentation Station**"...directed towards as close as possible to live entry.

I worked very closely with Jonathan Reams (who will be on the this Friday's Monthly Research Group call) on a number of narratives projects, and he has since ran with, and further developed the narratives software I developed...perhaps another possible collaborator, who can add a key 'developmental psychology' metrics lens to such work.
